kv/kv.go

```diff
@@ -205,7 +205,7 @@ type Response interface {
 	// Next returns a resultSubset from a single storage unit.
 	// When full result set is returned, nil is returned.
 	// TODO: Find a better interface for resultSubset that can avoid allocation and reuse bytes.
-	Next(ctx context.Context) (resultSubset []byte, err error)
+	Next(ctx context.Context) (resultSubset []byte, startKey Key, err error)
```

Comment:
Why change the interface?

Reply:
Because we need to locate the scan range that this result subset corresponding to.